.yamato/project-pack.yml

```diff
@@ -14,4 +14,31 @@ pack_{{ project.name }}:
     packages:
       paths:
         - "upm-ci~/packages/**/*"
+{% endfor -%}
+
+# Pack minimal project with different versions of dependencies
```

Comment:
Pack minimal project with updated dependencies (UTP 2.0 in this case). This is then used by upm-ci tests in the pipeline